.github/workflows/stats_lambda.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow executes a Lamdba function that recording metrics for each successfully merged PR.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/marketplace/actions/invoke-aws-lambda
+
+name: Invoke runTheNumbers lambda functions
```

Comment:
I'd prefer something descriptive for this name that doesn't require knowing what 'runTheNumbers lambda functions' means for anyone finding this file and wondering what it does.